cmd/rook/ceph/osd.go

```diff
@@ -59,13 +60,22 @@ var filestoreDeviceCmd = &cobra.Command{
 	Short:  "Runs the ceph daemon for a filestore device",
 	Hidden: true,
 }
+var osdStartCmd = &cobra.Command{
+	Use:    "start",
+	Short:  "Starts the osd daemon", // OSDs that were provisioned by ceph-volume
+	Hidden: true,
+}
 var (
 	osdDataDeviceFilter string
 	ownerRefID          string
 	mountSourcePath     string
 	mountPath           string
 	osdID               int
 	copyBinariesPath    string
+	bluestoreOSD        bool
```

Comment:
Should we have something more generic such as `osdObjectStore`? And then, fill it with either `filestore` or `bluestore` and maybe something else in the future. We can have `osdObjectStore` default to `bluestore`.

Reply:
The bool is sprinkled in enough places that I don't want to touch it for this release, but I'll consider with future changes. Is there something else discussed beyond bluestore? Or this is just to generalize in case it happened?

Reply:
Oh I see so it does not come from this PR. Well, Filestore will stay around, it seems it has a use case. Also, there might be something new in the future. I guess having 2 stores already is enough to make it it's own `osdObjectStore` variable.